spec/features/staff/beis_users_can_view_organisations_spec.rb

```diff
@@ -24,33 +58,28 @@
       context "the role is 'delivery_partners'" do
         let(:role) { "delivery_partners" }
 
-        scenario "it lists delivery partner organisations" do
-          expect(page).to have_content(t("page_title.organisation.index"))
+        include_examples "lists delivery partner organisations"
 
-          delivery_partner_organisations.each do |organisation|
-            expect(page).to have_content(organisation.name)
+        context "it allows toggling to view matched effort providers" do
```

Comment:
This is super picky and feel free to disagree/ignore, but I think this would read better as something like:

> when viewing the matched effort providers tab…

Reply:
Good point, it's a context after all!